src/DynamoCoreWpf/ViewModels/RunSettingsViewModel.cs

```diff
@@ -154,14 +154,43 @@ public bool RunButtonEnabled
                     Model.RunType != RunType.Periodic;
             }
         }
+        public bool RunTypesEnabled
```

Comment:
I believe these are bind to view so has to be public? If yes, then please add comments for public properties

Reply:
@RobertGlobant20 yes, this variable is binded to the ComboBox.IsEnabled then cannot be private or internal, then I just added comments.
https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047